validation-test/compiler_crashers_2_fixed/sr11624.swift

```diff
@@ -0,0 +1,20 @@
+// This source file is part of the Swift.org open source project
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+
+// RUN: not %target-swift-frontend %s -O -whole-module-optimization
```

Comment:
Does WMO make a difference for this test?

Reply:
Yes, it only crashes with WMO.

Reply:
Crashes for me with just `-O` as well. Also, I think the above invocation is not correct - it should be `not %target-swift-frontend -O -whole-module-optimization %s -emit-sil`.